vaadin-iron-list-flow-parent/vaadin-iron-list-testbench/src/main/java/com/vaadin/flow/component/ironlist/testbench/IronListElement.java

```diff
@@ -40,7 +40,7 @@ public void scrollToRow(int rowIndex) {
      * @return the index of the first visible row
      */
     public int getFirstVisibleRowIndex() {
-        return getPropertyInteger("_firstVisibleIndex");
```

Comment:
These were changed by accident [here](https://github.com/vaadin/flow-components/pull/978/files#diff-6cc3d2274052d9f57d0557c0391d78542c16a239afcda2c057995b911a036625L43). There were no tests using the APIs so this wasn't caught automatically :(